src/editpage/components/assistance/SettingsModal.vue

```diff
@@ -174,6 +191,17 @@ export default {
       this.updateConfigurationString();
     },
 
+    exportLogs: function() {
+      const {dialog} = require('electron').remote;
+      const options = {defaultPath: 'logs.zip'};
+      const targetDir = dialog.showSaveDialogSync(options);
+      const electron = require('electron');
+      const sourceDir = (electron.app || electron.remote.app).getPath(
+          'userData') + '\\activity-logs';
+      const zipper = require('zip-local');
+      zipper.sync.zip(sourceDir).compress().save(targetDir + '.zip');
```

Comment:
Does `.sync` mean we are doing this synchronously? I think doing it asynchronously is fine and might keep the app more responsive.

Also can any of the steps in between fail?

Reply:
I'll change it to asynchronous zipping. As far as I know, the steps shouldn't fail, but I can add something in case an error is caught.